exampleSite/hugo.toml

```diff
@@ -4,15 +4,23 @@ baseURL = 'https://hba.sid.one/'
 # This is what goes in <html lang="">
 languageCode = 'en-us'
 
-theme = "hugo-blog-awesome"
```

Comment:
Could you please share, why this line is no longer required?

Reply:
> Could you please share, why this line is no longer required?

Because the theme is now pulled in as module.

For pulling in the theme as module, we can either write:
```
theme = "github.com/hugo-sid/hugo-blog-awesome"
```
or
```
[module]
  [[module.imports]]
    path = "github.com/hugo-sid/hugo-blog-awesome"
```

I decided to go for the second choice.

As a reaction on your question/comment, I added the line `theme = "hugo-blog-awesome` again, now as comment and added some comment lines to make the two options (traditional vs. hugo module) clearer for potential user of your theme. Does that meet your expectations?

Reply:
Hi Andreas, thank you so much for the clarifications.

Thank you for providing two options (traditional vs. Hugo module).

Could you please set the traditional way to be the default? I feel it has better developer experience (DX).